src/pages/blog/proxy-capabilities.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Microlink Proxy: How it works'
+image: 'https://i.imgur.com/Ckf9Tau.png'
+date: '2021-06-07'
+---
+
+All [Microlink Pro](#pricing) plans come with a built-in proxy layer called that does automatic proxy resolution & rotation based on the input URL.
+
+## What's wrong with URLs
+
+Internet is wild place, where every URL is different. That's specially an issue when you are doing things at scale.
+
+![](https://i.imgur.com/pVPDpao.png)
+
+<Figcaption>
+  {'A '}
+  <Link href='https://github.com/microlinkhq/metascraper/issues/417' children='metascraper issue' /> facing with fetching problems.
+</Figcaption>
+
+When you are surfing the net and visit a website, there are situations you can't handle at all:
+
+- The are geographic location restrictions (e.g., visiting a site from China).
+- Your IP address is blacklisted (e.g., performing request from a data center).
+- You have to validate your identity (e.g., filling a CAPTCHA).
+
+You can quickly face with these problems when you send enough traffic to popular sites, like Instagram or Bloomberg, making it indeterministic to get a successful response in a consistent way.
```

Comment:
"You can quickly face ~~with~~ these problems when you send enough traffic to popular sites, like Instagram or Bloomberg, which, in turn, makes getting a successful response inconsistent. ~~indeterministic  to get a successful response in a consistent way.~~

^
This one is a bit tricky, but I think this is a bit clearer...